src/Haskell/Ide/Engine/Version.hs

```diff
@@ -45,7 +45,7 @@ getProjectGhcVersion crdl = do
   if isStackCradle crdl && isStackInstalled
     then do
       L.infoM "hie" "Using stack GHC version"
-      catch (tryCommand "stack ghc -- --numeric-version") $ \e -> do
+      catch (last . lines <$> tryCommand "stack ghc -- --numeric-version") $ \e -> do
```

Comment:
Stack gives a warning we need to ignore.
```bash
$ stack ghc -- --numeric-version
Stack has not been tested with GHC versions above 8.6, and using 8.8.1, this may fail
Stack has not been tested with Cabal versions above 2.4, but version 3.0.0.0 was found, this may fail
8.8.1

```

Reply:
Those are printed to stderr, though, iirc.

Reply:
It doesnt matter though, since I remove the function anyways in #1496

Reply:
Your right, I added it when debugging 8.8.1 stack ghc version checking. It must have been something else, that fixed it.

Reply:
Can we just filter this to only read in stdout, not stderr?

Reply:
It already does. This change is likely to be a noop.